### ISSUE TITLE
Thank you for visiting out my profile.

### ISSUE THREAD

**Issue by mahshidy** [maryamyasaei/Personal-Website-#14]
I think you should drop "out". 
Thank you for visiting my profile.

Also

 _If you would like to get into contact with me, Please Email Me._
"get into" means influencing, or taking control of someone. This is not what you had in mind I think. Maybe rephrase it with:
_Email me, if you would like to get in touch with me._

**Comment by maryamyasaei** [comment]
in khob hamon jomle template bod, dige avaz nakardam dige ....
